source/reference/mongosync/mongosync-behavior.txt

```diff
@@ -140,16 +140,16 @@ Consistency
 ~~~~~~~~~~~
 
 ``mongosync`` supports :term:`eventual consistency` on the destination
-cluster. Read Consistency is not guaranteed on the destination cluster,
-the source and destination clusters may differ at a given point in
-time. However, if new writes are halted on the source cluster, the
-source and destination clusters will eventually hold the same data.
+cluster. Read consistency is not guaranteed on the destination cluster until 
+commit. Before committing, the source and destination clusters may differ at a 
+given point in time. However, if new writes are halted on the source cluster, 
```

Comment:
I think we should add "upon commit" to "eventually hold the same data" CC @rohshar

Reply:
That is fine with me. I think the idea is that the documents should eventually match if no new writes are happening on the source, even before commit, but other configuration may not. It is reasonable not to want to publicize this to clients.

Reply:
updated to just remove the sentence.